examples/simple/server.py

```diff
@@ -8,17 +8,19 @@
 Simple echo server, using nonblocking I/O
 """
 
-from OpenSSL import SSL
+from OpenSSL import SSL, crypto
 import sys, os, select, socket
 
 
 def verify_cb(conn, cert, errnum, depth, ok):
     # This obviously has to be updated
-    print 'Got certificate: %s' % cert.get_subject()
+    certsubject = crypto.X509Name(cert.get_subject())
+    commonname = certsubject.commonName
+    print(('Got certificate: ' + commonname))
```

Comment:
I should have made a comment to clarify :).

This is what you get on master:
`Got certificate: <X509Name object '/CN=Certificate Authority'>`
`Got certificate: <X509Name object '/CN=Simple Server'>`

This is what you get with that:
`Got certificate: Certificate Authority`
`Got certificate: Simple Server`

I think that is the direction the orig author was going in when he said: "# This obviously has to be updated".  I can remove the comment if you agree.  I suppose there are less spoofable things that we could use to identify the certs in addition to the CN, like the fingerprint.